app/controllers/api/edge/comments_controller.rb

```diff
@@ -2,7 +2,7 @@ class Api::Edge::CommentsController < Api::CommentsController
   private
 
   def update_mobile_user_identity
```

Comment:
Is this method needed here at all? Parent class defines this method which looks very similar (using `dig` instead of `fetch` on params). Is failing with `KeyError` the desired behavior in edge api and in normal not?